app.js

```diff
@@ -0,0 +1,140 @@
+const express = require("express")
+const cors = require("cors")
+const bodyParser = require("body-parser")
+const time = require("express-timestamp")
+
+
+const port = 3000
+
+const classRouter = require('./Class.js')
+const StudentRouter = require('./Student.js')
+const SchoolRouter = require("./School.js")
+
+const app = express()
+app.use(time.init);
+
+app.use(cors())
+app.use(bodyParser.urlencoded({
+    extended: false
+}))
+app.use(bodyParser.json())
+
+const School = require("./School.js")
+const Student = require("./Student.js")
+// const Classes = require("./Class.js")
+
+let mySchool = new School();
+
+
+app.get("/", (req, res) => {
+    res.json(newSchool);
+});
+
+
+// adding a class 
+
+// app.post("/class", (req, res) => {
+//             if (!mySchool.addClass(req.body.name, req.body.teacher)) {
+//                 res.json({
+//                     "error": "Please fill out all the information or Class already exists",
+//                     "timestamp": req.timestamp
+//                 })
+//             } else {
+//                 res.json({
+//                         "class": {
+//                             "name": req.body.name,
+//                             "teacher": req.body.teacher,
+//                             "students": []
+            
+//                         })
+//                 )
+//         })
+
+            app.post("/class", (req, res) => {
+                try {
+                    // let addNewClass = mySchool.addClass
+                    mySchool.addClass(req.body.name, req.body.teacher);
```

Comment:
Assign this to a variable. Let's say `newClass`. So `let newClass = mySchool.addClass(req.body.name, req.body.teacher)`, that way the return value of your `addClass` function in your School.js file will be assigned to the `newClass` variable and you can now pass that in your response.